### PR TITLE
SYMCO-87 audio works on feedback pages now

### PR DESCRIPTION
**Pull request by devin-w-hill**
very small changes. do a template, check the feedback and make sure the audio plays (both researcher and student pages).

### CODE REVIEW

File: simcon_project/templates/feedback/view_feedback.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h3> Response {{node.position_in_sequence}}</h3>
         </div>
         <div class="row">
             <div class="col-sm-4">
-                <audio src="{% static node.audio_response %}" controls></audio>
+                <audio src="{% get_media_prefix %}{{ node.audio_response }}" controls></audio>
```
Comment (ericharper1) [inline simcon_project/templates/feedback/view_feedback.html:50]: Wait this works? I thought it had to be node.audio_response.url

File: simcon_project/simcon_project/settings.py [hunk 2]
```diff
@@ -128,7 +128,6 @@
 # Email
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_USE_TLS = True
```
Comment (ericharper1) [inline simcon_project/simcon_project/settings.py:131]: What dat line do?

**Review by ericharper1 — APPROVED** [review]
Audio is there!

**Review by rosasck — APPROVED** [review]
Happpyyyyyyyyynessss

**Pull request merged**